hedera-node/hedera-app/src/xtest/java/contract/WipeXTest.java

```diff
@@ -63,9 +74,15 @@
  * <ol>
  *     <li>Wipe {@code ERC721_TOKEN} serial 1234 from Owner's account and verify successful operation</li>*
  *     <li>Wipe 10 {@code ERC20_TOKEN} Owner's account via wipeTokenAccountV1 and verify successful operation</li>*
- *     <li>Wipe 10 {@code ERC20_TOKEN} Owner's account via wipeTokenAccountV2 and verify successful operation</li>*
  *     <li>Via {@code assertExpectedAccounts} verify that owner's nft supply was decreased by 1.</li>*
  *     <li>Via {@code assertExpectedTokenRelations} verify that owner's token balance was decreased by 20.</li>*
```

Comment:
These assertions are done after the Scenario Operations, so we can move the comments to the bottom to keep the chronological order.

Reply:
Done. This was due to merge.